src/device-state/device-config.ts

```diff
@@ -524,7 +534,12 @@ async function getBackendSteps(
 	return [
 		// All backend steps require a reboot
 		...(steps.length > 0
-			? [{ action: 'setRebootBreadcrumb' } as ConfigStep]
+			? [
+					{
+						action: 'setRebootBreadcrumb',
+						target: { bootConfigChanged: 'true' },
```

Comment:
Why a string boolean?

Reply:
I think is just because of the existing types of the actions on device-config, I'll check if it makes sense to refactor